collections/_article/release-candidate-godot-4-0-rc-1.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Release candidate: Godot 4.0 RC 1"
+excerpt: "The wait is almost over! With Godot 4.0 coming close to stable, we finalize our efforts to fix the remaining critical issues and add the last coat of polish with the first Release Candidate."
+categories: ["pre-release"]
+author: Yuri Sizov
+image: /storage/blog/covers/release-candidate-godot-4-0-rc-1.jpg
+image_caption_title: "Sophia's Rainbow"
+image_caption_description: "An upcoming demo by GDQuest"
+date: 2023-02-08 13:00:00
+---
+
+This has been a long road, but the exciting times are upon us! Thanks to our excellent contributors and our brave volunteer beta testers we are reaching the biggest milestone in the history of Godot so far. We are about to release Godot 4.0 stable. We are very proud of how it has shaped, about features and enhancements that has been implemented, and most of all, we are proud to have collected such a brilliant team of talented individuals who have carried this major release on their wide and mighty shoulders.
+
+But before we can cut the ribbon and break out the champagne, there are still a few preview releases we must go through, to establish, with your help, that we are truly ready. What you can expect from the next couple of weeks is the same cadence of official builds, aiming to minimize the iteration time between fixing bugs and checking for regressions. We will no longer be making any breaking changes, or include further enhancements. New features will have to [wait for Godot 4.1 later this year](/article/release-management-4-0-and-beyond). Instead, we will be coming down on the last critical issues we want to be resolved before we ship the first stable release of Godot 4.
```

Comment:
```suggestion
But before we can cut the ribbon and break out the champagne, there are still a few preview releases we must go through, to establish, with your help, that we are truly ready. What you can expect from the next couple of weeks is the same cadence of official builds, aiming to minimize the iteration time between fixing bugs and checking for regressions. We will no longer be making any breaking changes or including further enhancements. New features will have to [wait for Godot 4.1 later this year](/article/release-management-4-0-and-beyond). Instead, we will be coming down on the last critical issues we want to be resolved before we ship the first stable release of Godot 4.
```

Reply:
I prefer the original version, it's clearer IMO.

It's "[We will no longer] be making..." and "[We will no longer] include..." bound together with "or".
Your version would imply the second one to be "[We will no longer be] including..." which sounds quite awkward I think.

Reply:
Could also be "We will no longer make ... or include ...".